schema/all-zones-requests.json

```diff
@@ -233,6 +233,17 @@
         "zone_type"
       ],
       "properties": {
+        "filesystem_pool": {
```

Comment:
The schemas here **are** changing, but they should be adding a nullable field.

My expectation is that this will be backwards-compatible with on-disk storage. If you, dear reviewer, think otherwise, please comment! My intent with leaving the field optional was to avoid breakages.